### ISSUE TITLE
Exception when choosing developer menu option

### ISSUE THREAD

**Issue by mbrandis** [wallabag/wallabag#2501]
### Issue details

I upgraded wallabag from 2.0.8 to 2.1.1. I followed the documentation also upgrading the database.

Everything seemed to work fine. But as soon as I select "Developer" in the menu, I get the following error message in the browser:

500: Internal Server Error

An exception occurred while executing 'SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5, t0.name AS name_6 FROM wallabag_oauth2_clients t0': SQLSTATE[HY000]: General error: 1 no such column: t0.name

### Environment

* Version: 2.1.1
* Using the shared host package
* Version: 2.0.8 worked
* php version: 5.6.27
* OS: Debian stable
* Dedicated, but using shared package
* sqlite

### Steps to reproduce/test case

Upgrade from 2.0.8 to 2.1.1, login, choose "Developer" menu option.

**Comment by nicosomb** [comment]
Hello @mbrandis! 

Did you run the SQL queries to upgrade your database (see http://doc.wallabag.org/en/master/user/upgrade-2.0.x-2.1.1.html#upgrade-on-a-shared-hosting)?

**Comment by mbrandis** [comment]
Yes, I did. I used the sqlite3 command line client for that and copy and pasted the queries.

**Comment by j0k3r** [comment]
Could you show the table structure of `wallabag_oauth2_clients`?
I guess the upgrade using the MySQL SQL from the doc on SQLite doesn't work well.

**Comment by mbrandis** [comment]
`sqlite> PRAGMA table_info(wallabag_oauth2_clients);
0|id|INTEGER|1||1
1|random_id|VARCHAR(255)|1||0
2|redirect_uris|CLOB|1||0
3|secret|VARCHAR(255)|1||0
4|allowed_grant_types|CLOB|1||0
`

**Comment by j0k3r** [comment]
That's why I thought. The `name` wasn't created. Could you run:

```sql
ALTER TABLE `wallabag_oauth2_clients` ADD name longtext DEFAULT NULL;
```

**Comment by nicosomb** [comment]
Argh, I'm not sure that `longtext` exists for SQLite. 

```
CREATE TABLE wallabag_oauth2_clients (id INTEGER NOT NULL, random_id VARCHAR(255) NOT NULL, redirect_uris CLOB NOT NULL, secret VARCHAR(255) NOT NULL, allowed_grant_types CLOB NOT NULL, name CLOB DEFAULT NULL, PRIMARY KEY(id))
```

Here is my CREATE query for SQLite.

**Comment by j0k3r** [comment]
When I fixed migrations for the 2.2, the update run smoothly with: https://github.com/wallabag/wallabag/blob/8e58be9fb62615cdd8f4b55ad0981f57641bf0ab/app/DoctrineMigrations/Version20160812120952.php

**Comment by mbrandis** [comment]
It is working with the first version of the ALTER TABLE command. 

Thank you!
